src/arch/x86/x86_64/src/lib.rs

```diff
@@ -6,9 +6,11 @@
 
 const PAGE_SIZE: usize = 4096;
 pub mod acpi;
+pub mod amdromehsmp;
 pub mod bzimage;
 pub mod consts;
 pub mod ioport;
+pub mod pci;
```

Comment:
probably same for this. If pci is here it's a good bet our structure needs to be reworked.

Reply:
Moved to soc rome.